data/cluster-options.yaml

```diff
@@ -26,12 +26,12 @@ CC-Total_Answered:
   warning-level-low: "4"
   worst-level-high: "20"
   worst-level-low: "0"
-ClickToDial2:
-  label: Dial
-  url: http://esx36-ip21-174.gws.genesys.com/sample/sample.html
-CustomAPI:
+CustomLocalAPI:
   label: API
-  url: https://localhost:3000/sample/service-client-api/sample.html
+  url: https://localhost:8080/scapi/sample.html
+ScapiRemoteGithub:
```

Comment:
Could be renamed by SCAPISampleRemoteGitHub

Reply:
Finally, "RemoteSCAPISample" is fine, no need to explicitly use "Github" in the name.